<launch>
    <arg name="param_topic" default="/offroad_learning/params" />
    <arg name="rb_topic" default="/offroad_learning/rb_data" />

    <node pkg="offroad_learning" type="training_node" name="training" output="screen">
        <env name="XLA_PYTHON_CLIENT_MEM_FRACTION" value="0.7" />
        <env name="LD_LIBRARY_PATH" value="$(env LD_LIBRARY_PATH):$(env HOME)/anaconda3/lib" />
        <param name="~param_topic" type="string" value="$(arg param_topic)" />
        <param name="~rb_topic" type="string" value="$(arg rb_topic)" />
    </node>
</launch>